en/flight_modes/takeoff.md

```diff
@@ -28,50 +28,100 @@ Parameter | Description
 <span id="MIS_TAKEOFF_ALT"></span>[MIS_TAKEOFF_ALT](../advanced_config/parameter_reference.md#MIS_TAKEOFF_ALT) | Target altitude during takeoff (default: 2.5m)
 <span id="MPC_TKO_SPEED"></span>[MPC_TKO_SPEED](../advanced_config/parameter_reference.md#MPC_TKO_SPEED) | Speed of ascent (default: 1.5m/s)
 <span id="COM_RC_OVERRIDE"></span>[COM_RC_OVERRIDE](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) | Controls whether stick movement on a multicopter (or VTOL in MC mode) causes a mode change to [Position mode](../flight_modes/position_mc.md). This can be separately enabled for auto modes and for offboard mode, and is enabled in auto modes by default.
-<span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled).
+<span id="COM_RC_STICK_OV"></span>[COM_RC_STICK_OV](../advanced_config/parameter_reference.md#COM_RC_STICK_OV) | The amount of stick movement that causes a transition to [Position mode](../flight_modes/position_mc.md) (if [COM_RC_OVERRIDE](#COM_RC_OVERRIDE) is enabled)
 
-<span id="fixed_wing"></span>
-## Fixed Wing (FW)
+<a id="fixed_wing"></a>
+## Fixed-wing (FW)
+
+Automatic takeoff has two modalities: *catapult/hand-launch* or *runway takeoff*.

```

Comment:
@sfuhrer I'm going to merge this now - I think it is OK.

My main concern is that we are mixing the takeoff mode and mission mode takeoff, as discussed. I think we could do a better job here, but maybe as a post process.

What we could do is:
1. Duplicate just this section in mission mode FW takeoff, and make it mission mode specific. 
2. Then reference the catapult and runway sections in takeoff mode for the "common stuff".
3. Remove the mission mode specific stuff from here.

We might then put the stuff currently in mission mode into the "Planning" doc. 

Let me know what you think?

Reply:
So where would the catapult and runway sections live? In the mission or the flight modes docs, or on a new, neutral ground?